doc/api/tls.md

```diff
@@ -876,6 +876,30 @@ socket.on('end', () => {
 });
 ```
 
+Passing in `net.Socket` will start a new instance.
```

Comment:
Good point. In the original issue, the user was attempting to do this because he didn't understand that if you pass in a net.Socket, you start the net.Socket.:

`const tlsSocket = new TlsSocket(new NetSocket());`

I just wanted to make sure that that was clearly stated; but if it's clear in the subsequent lines, I'll just take it out 😄